src/EventListener/InitializeSystemListener.php

```diff
@@ -325,12 +325,12 @@ private function validateInstallation(Request $request = null)
 
         // Show the "insecure document root" message
         if (!in_array($request->getClientIp(), ['127.0.0.1', 'fe80::1', '::1']) && '/web' === substr($request->getBasePath(), -4)) {
-            throw new InsecureInstallationHttpException(null, 'Your installation is not secure. Please set the document root to the <code>/web</code> subfolder.');
+            throw new InsecureInstallationException('Your installation is not secure. Please set the document root to the <code>/web</code> subfolder.');
```

Comment:
We now also should remove the `<code>` tag from the message as it is not an http exception anymore.